test/test_va_api_get_create_config.cpp

```diff
@@ -54,19 +56,70 @@ TEST_P(VAAPIGetCreateConfig, CreateConfigWithAttributes)
         doQueryConfigEntrypoints(profile);
         if (doFindEntrypointInList(entrypoint)) {
             // profile and entrypoint are supported
-            doFillConfigAttribList();
-            doGetConfigAttributes(profile, entrypoint);
-            doCreateConfigWithAttrib(profile, entrypoint);
-            doDestroyConfig();
+            ConfigAttributes attribs;
+            getConfigAttributes(profile, entrypoint, attribs);
+
+            // create config with each individual supported attribute
+            for (const auto& attrib : attribs) {
+                const auto match = g_vaConfigAttribBitMasks.find(attrib.type);
+                if (match != g_vaConfigAttribBitMasks.end()) {
+                    // it's a bitfield attribute
+                    uint32_t bitfield(0);
+                    const BitMasks& masks = match->second;
+                    for (const auto mask : masks) { // for all bitmasks
+                        const bool isSet((attrib.value & mask) == mask);
```

Comment:
Some flags are not bitmask in va.h, e.g. VA_ENC_PACKED_HEADER_NONE is defined as '0' in va.h.

Reply:
Yes, thanks for catching this.  I'll remove VA_ENC_PACKED_HEADER_NONE and VA_ENC_INTERLACED_NONE from the bitmask containers.